src/style.css

```
@@ -1,3 +1,83 @@
+
 body {
   margin: 0;
+padding: 0;
+background-color: #fff;
+border: 0;
+font-family: Roboto, sans-serif;
 }
+
+header {
+display: flex;
+justify-content: space-between;
+align-items: center;
+padding: 0 50px;
+}
+
+.logo {
+display: block;
+line-height: 0;
+}
+
+.header {
+box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+background: #FFF;
+
+}
+
+.nav-link {
+text-decoration: none;
+display: flex;
+align-items: center;
+color: black;
+position: relative;
+text-transform: uppercase;
+padding: 23px 0;
+  }
+
+.nav-list {
+  margin: 0;
+display: flex;
+list-style-type: none;
+font-family: Roboto, sans-serif;
+font-size: 12px;
+font-style: normal;
+font-weight: 500;
+line-height: normal;
+text-transform: uppercase;
+align-items: center;
+justify-content: flex-end;
+padding: 0;
+}
+
+.nav-item {
+margin-left: 20px;
+}
+
+.nav-item:first-child {
+margin-left: 0;
+}
+
+.nav-link:hover {
+color: #00ACDC;
+}
+
+.is-active::after{
+position:absolute;
```

please, work with indents all over the page
```suggestion
position: absolute;
```